### ISSUE TITLE
node-minify/uglify-es to sourceMap

### ISSUE THREAD

**Issue by w4567892015** [srod/node-minify#685]
Hi ! How can I use it to save sourceMap on file ?

**Comment by w4567892015** [comment]
https://www.npmjs.com/package/uglify-es#source-map-options

**Comment by srod** [comment]
Hi, you have to pass an option like that:
```jsminify({
  compressor: uglifyES,
  input: 'foo.js',
  output: 'bar.js',
  options: {
    sourceMap: {
      url: "bar.js.map"
    }
  }
});```

**Comment by w4567892015** [comment]
Thanks! But it have some problem, the your url is loacl path to save file, But i will url to remote path > http://127.0.0.1:3000/bar.js.map, Not origin sevrer send the file, And the sourceMap source array is "0", not origin source path.

**Comment by zhangmx** [comment]
should it be same as 'settings.output' with '.map'?

**Comment by w4567892015** [comment]
> should it be same as 'settings.output' with '.map'?

Yes, but map source array is "0",

**Comment by zhangmx** [comment]
@w4567892015 i made new change in my fork, but i don't get how to pull a new request.

**Comment by srod** [comment]
Thanks guys, I will see that as soon as possible!

**Comment by w4567892015** [comment]
okay, good job!

**Comment by srod** [comment]
I've released 4.1.2.
